dataactbroker/fileRoutes.py

```diff
@@ -174,6 +176,20 @@ def post_submission_narratives(submission):
     def post_submission_report_url(submission, warning, file_type, cross_type):
         return submission_report_url(submission, bool(warning), file_type, cross_type)
 
+    @app.route("/v1/delete_submission/", methods=['POST'])
+    @convert_to_submission_id
+    @requires_submission_perms('writer')
+    def delete_submission(submission):
+        """ Deletes all data associated with the specified submission
+        NOTE: THERE IS NO WAY TO UNDO THIS """
+
+        sess = GlobalDB.db().session
+        sess.query(Submission).filter(Submission.submission_id == submission.submission_id).delete(
+            synchronize_session=False)
+        sess.expire_all()
```

Comment:
Meant to ask this before, does this expire all things with a relationship to this particular submission? For example, if the session has an awardfinancial object related to the submission, will it think it isn't expired? (don't know if this is relevant to how we do anything, but worth double-checking)